packages/opds-web-client/src/components/BookCover.tsx

```diff
@@ -53,7 +53,9 @@ export default class BookCover extends React.Component<BookCoverProps, BookCover
     return (
       <div className="auto-book-cover" style={{ backgroundColor: bgColor }}>
         <div className="title" style={{ fontSize: titleFontSize }}>{ title }</div>
-        <div className="authors" style={{ fontSize: authorFontSize }}>By { authors.join(", ") }</div>
+        { authors.length &&
+          <div className="authors" style={{ fontSize: authorFontSize }}>By { authors.join(", ") }</div>
+        }
```

Comment:
Just out of curiosity, what's the authorFontSize thing?  Is the font size in which authors' names are displayed determined by how many authors there are...or...?

Reply:
Yes, that seems to be correct. The bigger the word count the smaller the font size.

Reply:
Yes, this is for automatically generating book covers for books that don't have them, so the title and authors need to fit in a fixed size cover. The longer the text is the smaller it needs to be.

Reply:
This is great. We needed something like this for New Arrivals a while ago.